src/cnaas_nms/confpush/sync_devices.py

```diff
@@ -20,8 +20,10 @@
 from cnaas_nms.db.git import RepoStructureException
 from cnaas_nms.confpush.nornir_helper import NornirJobResult
 from cnaas_nms.scheduler.wrapper import job_wrapper
+from cnaas_nms.scheduler.scheduler import Scheduler
 
 logger = get_logger()
+AUTOPUSH_MAX_SCORE = 10
```

Comment:
Should this be configurable in some way?

Reply:
I can see two options
1. We fix this value and document it clearly, and if you want to tune what changes are allowed as autopush you tune the modifier values in text pattern matching or some other part of the score calculation. Maybe makes UI/UX more clear? But it's also less flexible, unless the pattern matching etc is also in a configurable file.
2. We allow this value to be changed in a config file, and we tune this instead of updating the score calculation algorithm. Maybe scores will be more consistent over time?
Maybe we need user input for this as well